flakes/patterns.txt

```diff
@@ -5,3 +5,4 @@ CITimeoutException: Waiting for IP on interface Ethernet1 failed.
 SIGNTASK : SignTool error : No certificates were found that met all the given criteria\.
 \[[[:digit:]]+-controller\]: FAILED.*Failed to connect to yum.dockerproject.org at port 443
 ERROR:zuul\.Repo:Unable to initialize repo for https://review\.opencontrail\.org/Juniper/[a-z-]+\s*$
+Get https://registry-1\.docker\.io/v2/:.*getsockopt: no route to host
```

Comment:
Maybe we should change `registry-1` to `registry-[[:digit:]]+`? From what I understand, there are multiple registry servers and you can be redirected to different one. (perhaps I'm wrong though)

Reply:
I would stay with `registry-1`, because I did not find any information about registries other than `registry-1`. Also DNS properly resolves only for `registry-1.docker.io` from what I checked and `registry-1` itself resolves to multiple IP addresses.

Reply:
But I'll add the flake description test file.